src/Sylius/Bundle/ShopBundle/Resources/views/_footer.html.twig

```diff
@@ -1,4 +1,5 @@
 <footer id="footer" class="ui inverted vertical footer segment">
+    {% include "@SyliusShop/Layout/Footer/Grid/_pre_footer.html.twig" %}
```

Comment:
I know that the Symfony recommendation is to use `snake_case` for Twig templates, but in Sylius we have been using `camelCase` for ages:
```suggestion
    {% include "@SyliusShop/Layout/Footer/Grid/_preFooter.html.twig" %}
```

Reply:
yeah You're right, but in this particular folder, all files (don't know why) are named that way, so I didn't want to break the convention

Reply:
😢